api/vm/query.ts

```diff
@@ -20,7 +20,8 @@ export function compile(query: string, projectDir?: string): string {
       projectDir,
       `(function() {
         const { session } = require("@dataform/core");
-        const ref = session.ref.bind(session);
+        const ref = session.resolve.bind(session);
```

Comment:
`ref` no?

Reply:
I renamed session.ref to session.resolve as that's all it's really doing.